meerk40t/core/webhelp.py

```diff
@@ -7,6 +7,7 @@
 DISCORD_MEERK40T = "https://discord.gg/6wxrJ2hn"
 MAKERS_FORUM_MEERK40T = "https://forum.makerforums.info/c/k40/meerk40t/120"
 IRC_CLIENT = "http://kiwiirc.com/client/irc.libera.chat/meerk40t"
+MEERK40T_FEATURE = "https://github.com/meerk40t/meerk40t/discussions/1318"
```

Comment:
This discussion is tiger asking about version or whatever, and seems like feature requests would be more just raise an issue. Also, I thought I got rid of the irc thing, since that's not a great way to contact us.